src/api/TwitarrAPI3Events.ts

```diff
@@ -0,0 +1,29 @@
+import TwitarrAPI3Module from "./TwitarrAPI3Module";
+
+export interface EventData {
+  description: string;
+  // Formatted in ISO 8601 format e.g. 2020-03-08T05:00:00.000Z
+  endTime: string;
+  eventID: string;
+  // e.g. "Official", "Gaming", "Shadow Event", etc.
+  eventType: string;
+  forum: string;
+  isFavorite: boolean;
+  location: string;
+  // Formatted in ISO 8601 format e.g. 2020-03-08T05:00:00.000Z
+  startTime: string;
+  title: string;
+  uid: string;
+}
+
+export default class TwitarrAPI3Events extends TwitarrAPI3Module {
+  getEvents = this.createAPIMethod<null, EventData[]>({
```

Comment:
If you want this to have no params, you can use `void` here.

```suggestion
  getEvents = this.createAPIMethod<void, EventData[]>({
```

Reply:
Oh yeah, duh. Thanks